addons/base_vat/views/res_config_settings_views.xml

```diff
@@ -6,22 +6,15 @@
         <field name="model">res.config.settings</field>
         <field name="inherit_id" ref="account.res_config_settings_view_form"/>
         <field name="arch" type="xml">
-            <div id="eu_service" position="after">
-                <div class="col-12 col-lg-6 o_setting_box"
+            <setting id="eu_service" position="after">
+                <setting
                     id="vies_service_setting"
+                    help="Verify VAT numbers using the European VIES service"
+                    company_specific="1"
```

Comment:
change attribute name to company_dependent (see formlabl.js &
and the attribute on fields python)

Reply:
retrieve this info on the fields_get ?